services/src/main/scala/cromwell/services/ServiceRegistryActor.scala

```diff
@@ -79,6 +87,16 @@ class ServiceRegistryActor(globalConfig: Config) extends Actor with ActorLogging
           log.error("Received ServiceRegistryMessage requesting service '{}' for which no service is configured.  Message: {}", msg.serviceName, msg)
           sender ! ServiceRegistryFailure(msg.serviceName)
       }
+    case meta: ServiceRegistryMetaRequest => meta match {
+      case RequestIoActorRef => ioActor match {
+        case Some(ref) => sender ! IoActorRef(ref)
+        case None => sender ! NoIoActorRefAvailable
+      }
+      case IoActorRef(ref) =>
+        ioActor foreach { existingIoActor =>
+          log.error(s"Programmer Error: More than one IoActor is trying to register itself in the service registry ($ref will replace $existingIoActor)") }
+        ioActor = Option(ref)
```

Comment:
Why replace the current `ref` with the one it received?

Reply:
Updated